### PR TITLE
Subaru: Add Impreza/Crosstrek 2020

### PR DESCRIPTION
**Pull request by martinl**
**Checklist**
- [x] added to README
- [x] test route added to [test_routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/test/test_routes.py)
- [x] route with openpilot: 8bf7e79a3ce64055|2021-05-24--09-36-27
- [x] route with stock system: 8bf7e79a3ce64055|2021-05-24--09-58-54

2020+ Crosstrek and Impreza require lower steering torque limit (1439) and different tuning than previous model years. Exceeding the steering torque limit causes Eyesight fault. 

Prerequisite: https://github.com/commaai/panda/pull/650
Routes provided by Discord user Crispin

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Left a comment  on the panda PR. Converting to draft for now.

**Comment by martinl** [comment]
I changed the panda PR, it includes only parts relevant to Impreza/Crosstrek 2020 now

**Comment by adeebshihadeh** [comment]
Is the scale the same as the other Subarus, i.e. is 1439 the same amount of torque on all Subarus? If so, no need for any panda changes. If not, we'll need to rent one and verify the safety and thresholds.

**Comment by martinl** [comment]
Yes, torque scale is same. I'll remove panda safetyParam from interface and close panda safety PR

File: selfdrive/test/test_routes.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
   HYUNDAI.ELANTRA_GT_I30,
   HYUNDAI.GENESIS_G90,
   HYUNDAI.KIA_OPTIMA_H,
+  SUBARU.IMPREZA_2020,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:24]:
```suggestion
```

**Comment by ClockeNessMnstr** [comment]
> Is the scale the same as the other Subarus, i.e. is 1439 the same amount of torque on all Subarus? If so, no need for any panda changes. If not, we'll need to rent one and verify the safety and thresholds.

No

**Pull request merged**